gh-util.user.js

```diff
@@ -150,7 +150,12 @@
             const url = window.location.href;
```

Comment:
```suggestion
            const url = window.location.pathname;
```

[Location.pathname](https://developer.mozilla.org/en-US/docs/Web/API/Location/pathname)
A string containing an initial '/' followed by the path of the URL, not including the query string or fragment.